docs/howto/connection/gcp.rst

```diff
@@ -95,4 +95,4 @@ Number of Retries
 
     .. code-block:: bash
 
-       google-cloud-platform://?extra__google_cloud_platform__key_path=%2Fkeys%2Fkey.json&extra__google_cloud_platform__scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&extra__google_cloud_platform__project=airflow&extra__google_cloud_platform__num_retries=5
+       export AIRFLOW_CONN_GOOGLE_CLOUD_DEFAULT='google-cloud-platform://?extra__google_cloud_platform__key_path=%2Fkeys%2Fkey.json&extra__google_cloud_platform__scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&extra__google_cloud_platform__project=airflow&extra__google_cloud_platform__num_retries=5'
```

Comment:
Check line 92. The example shown here is not of creating connection using environment variable but an example of how **components of the URI should be URL-encoded.**

Reply:
Update that line as well if your intention is to show this as an example of creating connection using an environment variable.

Reply:
Actually I wrote it initially and - @mik-laj is right: this is all about making the connection using the environment variable (see line 77). You do not need to do the same when defining the connection using database as no URL encoding is needed there.

Reply:
👍